src/mono/mono/component/marshal-ilgen.h

```diff
@@ -42,5 +58,8 @@ mono_emit_marshal_ilgen (EmitMarshalContext *m, int argnum, MonoType *t,
 
 void
 mono_marshal_ilgen_init_internal (void);
+
+void
+mono_marshal_ilgen_install_callbacks_mono (IlgenCallbacksToMono *callbacks);
```

Comment:
I'm not sure why this function needs a declaration here. It can be made static in `marshal-ilgen.c` and this declaration can be removed.